README.md

```diff
@@ -330,6 +330,11 @@ Then you can run this in the checkout root to reformat all Rust code:
 $ make rustfmt
 ```
 
+### Running elisp tests
+
+You can run the elisp test suite from the directory "test".
+Use `make` to run all tests. It's also possible to run single tests, e.g. `make src/data-tests`.
```

Comment:
Why not mention toplevel `make check` and `make check-maybe`? (Together with touching `.el` files to rerun with `check-maybe`?)

Reply:
I didn't know about these commands. Thanks.

Reply:
Can we also get a pointer to writing new tests? I tried to add a new file this morning but the make system claimed to not know about it.

Testing from Rust is a bunch of boiler plate to make LispObjects so I figured I'd write elisp tests.

Reply:
I don't think this is suitable for the walkthrough. Elisp tests need to be merged upstream and too much text can be confusing. But maybe we can put it in the README in the test directory ?